helm-chart/openj9-jitserver-chart/README.md

```diff
@@ -0,0 +1,228 @@
+    requiredDuringSchedulingIgnoredDuringExecution:
+        - key: kubernetes.io/hostname
+          operator: In
+          values:
+          - "example-node-name"
+```
+
+## Application and JITServer life cycle management
+
+### Verifying if the JITServer is deployed
+``` bash
+$ helm status jitserver-release
+NAME: jitserver-release
+LAST DEPLOYED: Wed Nov 18 18:41:07 2020
+NAMESPACE: helm-chart-test
+STATUS: deployed
+REVISION: 1
+NOTES:
+Welcome to OpenJ9 JITServer Helm Chart, the application has been deployed successfully.
+```
+
+``` bash
+$ helm test jitserver-release
+Pod jitserver-release-openj9-jitserver-chart-test-connection pending
+Pod jitserver-release-openj9-jitserver-chart-test-connection succeeded
+NAME: jitserver-release
+LAST DEPLOYED: Wed Nov 18 19:19:46 2020
+NAMESPACE: helm-chart-test
+STATUS: deployed
+REVISION: 1
+TEST SUITE:     jitserver-release-openj9-jitserver-chart-test-connection
+Last Started:   Wed Nov 18 19:41:14 2020
+Last Completed: Wed Nov 18 19:41:26 2020
+Phase:          Succeeded
+NOTES:
+Welcome to OpenJ9 JITServer Helm Chart, the application has been deployed successfully.
+```
+
+### Upgrade JITServer JDK upon application JDK upgrade.
+The JITServer instance must be compatible with the application JDK it serves. The easiest way to ensure this is using the same OpenJ9 release on both JITServer JDK and application JDK. Please follow [Step 1a](./README.md#a-verify-the-application-jdk-version-and-select-the-appropriate-jitserver-image) to find out application JDK build version before proceeding. 
+
+#### Scenario 1: All applications are being upgraded to a newer JDK version.
+If all applications are being upgraded to a newer version of OpenJ9 JDK, you may perform a `helm upgrade` to upgrade the JITServer instances to be compatible with your applications. Replace the `REPLACE_IMAGE_TAG` with the desired image tag of the JITServer to be deployed.
+```bash
+helm upgrade --set image.tag="REPLACE_IMAGE_TAG" jitserver-release openj9-jitserver-chart
+```
+
+#### Scenario 2: Only some applications are being upgraded to a newer version of JDK.
+If some applications are upgraded to a newer OpenJ9 JDK version while others are still using the older version of OpenJ9 JDK, you need to keep the existing JITServer and deploy another JITServer instance that is compatible with newer version of JDK. Replace `REPLACE_IMAGE_TAG` with the desired image tag to be deployed as a JITServer instance. 
+```bash
+helm install --set image.tag="REPLACE_IMAGE_TAG" jitserver-release-latest openj9-jitserver-chart
+```
+When all applications completed upgrades, and the older versions of the application are retired, you are safe to delete the older version of JITServer.
+``` bash
+helm delete jitserver-release
+```
+
+### Removing the JITServer
+Avoid removing the JITServer instance before applications terminate. There is a risk of application JVM has insufficient memory since it assumes JITServer handles the JIT Compilations.
```

Comment:
Rephrase "There is a risk of application JVM has insufficient memory since it assumes JITServer handles the JIT Compilations."
into
" If the memory limit for the container running the application JDK has been set based on the assumption that JITServer handles all JIT compilations, the application may experience a native out-of-memory scenario and be terminated"